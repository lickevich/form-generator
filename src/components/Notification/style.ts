import styled from "styled-components";

export const StyledNotification = styled.div`
  height: 90px;
  padding-top: 30px;
  color: orange;
`;
