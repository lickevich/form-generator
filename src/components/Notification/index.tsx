import React from "react";

import { StyledNotification } from "./style";
import { INotification } from "../../type";

const Notification: React.FC<INotification> = ({ message }) => {
  return <StyledNotification>{message}</StyledNotification>;
};

export default Notification;
