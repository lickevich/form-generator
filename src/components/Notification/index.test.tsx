import React from "react";
import { render } from "@testing-library/react";

import Notification from "./";

describe("Notification", () => {
  it("render the component", () => {
    const component = <Notification message="message_1" />;
    const { asFragment } = render(component);

    // @ts-ignore
    expect(asFragment(component)).toMatchSnapshot();
  });
});
