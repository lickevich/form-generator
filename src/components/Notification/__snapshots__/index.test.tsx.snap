// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Notification render the component 1`] = `
<DocumentFragment>
  <div
    class="sc-bdvvaa iucQhO"
  >
    message_1
  </div>
</DocumentFragment>
`;
